src/app/src/utils/time.ts

```diff
@@ -0,0 +1,20 @@
+export function timeSince(utcDateString: string): string {
```

Comment:
Nice util! I'm wondering tho if this is possible using dayjs or luxon? I'm considering just using dayjs instead of luxon because it is a lot smaller and does all the same things